src/main/java/de/tum/in/www1/artemis/repository/LectureRepository.java

```diff
@@ -44,4 +44,16 @@
             """)
     Optional<Lecture> findByIdWithStudentQuestionsAndLectureUnitsAndLearningGoals(@Param("lectureId") Long lectureId);
 
+    /**
+     * Returns the title of the lecture with the given id
+     *
+     * @param lectureId the id of the lecture
+     * @return the name/title of the lecture
+     */
+    @Query("""
+            SELECT l.title
+            FROM Lecture l
+            WHERE l.id = :lectureId
+            """)
+    String getLectureTitle(Long lectureId);
```

Comment:
what happens if the id cannot be found? does it return null? do we need to handle this case here?

Reply:
The method will return `null`. We don't need to handle that case. (See comment below.)